### PR TITLE
Add detail to README on development process

### PR DESCRIPTION
**Pull request by gene1wood**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,17 +48,32 @@ This cycle could be improved.
...
+#### 2 Commit your changes and deploy to auth-dev with CI
+
+1. Push your branch to your fork and create a PR with your new rule, requesting 
    a review of the PR.
-7. Someone reviews the PR, either suggesting changes or approving
-8. Merge the PR
-9. CI deploys the PR to auth0-dev
+2. Someone reviews the PR, either suggesting changes or approving
+3. Merge the PR to `master`
+4. CI deploys the PR to auth0-dev
```
Comment (floatingatoll) [inline README.md:76]: nit: `auth0-dev`

File: README.md [hunk 2]
```diff
@@ -65,17 +82,37 @@ Please note that for any large change (i.e. anything but a single rule change),
      [`uploader_rules.py`](https://github.com/mozilla-iam/auth0-ci/blob/master/uploader_rules.py)
      tool which is installed from the [`auth0-ci`](https://github.com/mozilla-iam/auth0-ci)
      project.
-10. Manually test again in auth0-dev to validate that the rule works. This is 
+5. Manually test again in auth0-dev to validate that the rule works. This is
```
Comment (floatingatoll) [inline README.md:85]: nit: `auth0-dev`

File: README.md [hunk 3]
```diff
@@ -65,17 +82,37 @@ Please note that for any large change (i.e. anything but a single rule change),
...
-13. [Test in prod](https://mana.mozilla.org/wiki/display/SECURITY/Auth0+manual+testing) to make sure everything works and rollback if it doesn't.  
+4. During change window, merge the PR. Now you have to manually run the 
+   CodeBuild job `auth0-deploy-prod` which will deploy the rules to the Auth0 
+   production tenant. You can do this using the AWS cli running 
+   `aws codebuild start-build --project-name auth0-deploy-prod`, or using the 
+   AWS UI console navigating to CodeBuild, choosing `auth0-deploy-prod`, 
+   pressing `Start build` and pressing again `Start build` in the next screen. 
+   Once the job finish successfully, all the rules should be uploaded to Auth0 
+   prod.
```
Comment (floatingatoll) [inline README.md:113]: Are there any functional changes in this section, or is it purely a reformat?

File: README.md [hunk 4]
```diff
@@ -65,17 +82,37 @@ Please note that for any large change (i.e. anything but a single rule change),
...
+   Auth0, you do the same kind of comparison we did in dev.
+   1. Checkout the `production` branch in your local repo
+   2. Run `uploader_rules.py` on the `auth0-deploy` rule directory
+      `./uploader_rules.py --dry-run --rules-dir ../auth0-deploy/rules`
+   3. Confirm that no differences are detected between your local code and what's
+      live in production Auth0
+2. Switch back to the `master` branch
+3. If testing validated that your new or changed rule is good, create a second 
+    PR from `master` to
     `production`, requesting review and referencing in the text of the PR the
```
Comment (floatingatoll) [inline README.md:101]: I think a paragraph rewrap was missed here, though I don't particularly mind at all.

File: README.md [hunk 5]
```diff
@@ -99,7 +136,7 @@ Next run the tests
 The Auth0 web UI where you can view and modify rules, for example at
 https://manage-dev.mozilla.auth0.com/dashboard/pi/auth-dev/rules
 has a jshint built in which isn't aware that Auth0 rules are run under
```
Comment (floatingatoll) [inline README.md:138]: I would suspect these days that the jshint is at Node 12 — is this still an applicable warning as written?

**Review by floatingatoll — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (gene1wood) [reply README.md:113]: No just formatting

File: README.md (hunk 5 above)
Comment (gene1wood) [reply README.md:138]: I'm not sure

**Comment by gene1wood** [comment]
Cool, I've fixed up those formatting and typo issues in 1f37afdf06aa35770e64307834596dedc4e6ad39

**Review by floatingatoll — APPROVED** [review]

**Pull request merged**